lib/CMakeLists.txt

```diff
@@ -458,29 +457,32 @@ foreach(name ecl_coarse_test
   add_executable(${name} ecl/tests/${name}.cpp)
   target_link_libraries(${name} ecl)
   target_include_directories(${name} PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/private-include)
-endforeach()
+endforeach(name)
```

Comment:
Miiinor style nitpick, but I personally dislike having anything inside the parens of block-end tags. Ie, `endforeach(name)` -> `endforeach()`. I'm up for discussing this.